nodemon.json

```diff
@@ -0,0 +1,7 @@
+{
+  "exec": "node --import 'data:text/javascript,import { register } from \"node:module\"; import { pathToFileURL } from \"node:url\"; register(\"ts-node/esm\", pathToFileURL(\"./\"));' src/index.ts",
```

Comment:
Oh, this looks horrible ..

Reply:
Yep but it's what the node people are suggesting https://github.com/nodejs/node/issues/51196